Common/src/linear_algebra/CSysMatrix.cpp

```diff
@@ -265,6 +265,9 @@ void CSysMatrix<ScalarType>::Initialize(unsigned long nPoint, unsigned long nPoi
     ILU_matrix = new ScalarType [nnz_ilu*nVar*nEqn];
     for (iVar = 0; iVar < nnz_ilu*nVar*nEqn; iVar++) ILU_matrix[iVar] = 0.0;
 
+    invM = new ScalarType [nPointDomain*nVar*nEqn];
```

Comment:
Addresses segfault found by @economon when using ILU with fill-in greater than 0.
Maybe in #790 one of the testcases can be modified to use this feature?